src/library/templates/library/library_bookshare_import.html

```diff
@@ -0,0 +1,85 @@
+{% extends "shared/base.html" %}
+{% load static figuration %}
+
+{% block bodyClass %}library{% endblock %}
+
+{% block title %}Add from Bookshare{% endblock %}
+
+{% block header %}
+{% include "shared/partial/site_header.html" %}
+{% endblock %}
+
+{% block content %}
+<main id="content" tabindex="-1">
+    <h1>Add from Bookshare</h1>
+    <div class="row">
+        <div class="col-md-9 col-lg-6">
+            <div class="box">
+                <h2>Step 3: Review and edit</h2>
+                <form method="post">
+                    <div class="form-group">
+                        {% csrf_token %}
+                        <p>Bookshare download:</p>
+                        <ul>
+                            <li>Title: {{ title.title }}</li>
+                            {% for author in title.clusive.authors %}
+                            <li>Author: {{ author }}</li>
+                            {% empty %}
+                            <li>No author</li>
+                            {% endfor %}
+                            <li>ISBN: {{ title.isbn13 }}</li>
+                            <li><img src="{{ title.clusive.thumbnail }}" alt="Cover of {{ title.title }}"</img></li>
+                            {% if title.clusive.more %}
+                            <li id="short_{{ title.bookshareId }}">
+                                Description: {{ title.clusive.synopsis }} &hellip;
+                                <button id="more_{{title.bookshareId}}" class="link-btn more-or-less">See more</button>
```

Comment:
Needs some disambiguation - suggest adding book title to button, , visually hidden but screen readers can see it.
Something like
`<button id="more_{{title.bookshareId}}" class="link-btn more-or-less">See more <span class="sr-only">about {{ title.title }}</span></button>`